### ISSUE TITLE
Python console throws exceptions upon BNDB load until view interaction

### ISSUE THREAD

**Issue by zackorndorff** [Vector35/binaryninja-api#3596]
**Version and Platform (required):**
- Binary Ninja Version: 3.2.3801-dev
- OS: macOS
- OS Version: 12.6 (Monterey)
- CPU Architecture M1

**Bug Description:**
If I open this BNDB and enter anything into the Python console, I get the following traceback.
If I open the BNDB and click on anything in the view, then enter anything into the Python console, I do not get a traceback.
So something's either serialized in an odd way or not initialized correctly, but setting a view position seems to fix it.

```
>>> 1
Traceback (most recent call last):
  File "/Applications/Binary Ninja.app/Contents/MacOS/plugins/../../Resources/python/binaryninja/scriptingprovider.py", line 748, in run
    self.update_locals()
  File "/Applications/Binary Ninja.app/Contents/MacOS/plugins/../../Resources/python/binaryninja/scriptingprovider.py", line 864, in update_locals
    if view_location is not None and view_location.isValid():
TypeError: unable to create object of type 'InstructionTextToken'
```

**Steps To Reproduce:**
Please provide all steps required to reproduce the behavior:

1. Open this particular BNDB (available upon request, its filename is `has-ftpd-binja-bug.bndb` and the  sha256sum is `13616f8ba5bc43bab574d492673ce9a68210a036754afb76481a7a58fdcccc04` (so I don't get them confused...)).
2. Do not click on anything in the open linear view.
3. Type `1` or `"hello"` or seemingly anything into the Python console
4. Observe traceback.

**Expected Behavior:**
I expected the Python expression to be evaluated and a result returned

**Comment by psifertex** [comment]
Please DM me the BNDB if you can on the BN slack or email it, thanks.

**Comment by psifertex** [comment]
FYI -- I apologize but I think I might have lost this repro file if it was shared. Can you re-send it please?

**Comment by zackorndorff** [comment]
No worries, it probably aged out of Slack history. Sent it to you on Slack, let me know if you need anything else!

**Comment by psifertex** [comment]
Thanks! I copied it to our internal slack this time so I wouldn't lose it. :-)

**Comment by psifertex** [comment]
Can you reproduce this issue still? On both latest stable and dev I'm not having experiencing the original issue at all. I tried with a fresh settings and no plugins just to double check as well.

**Comment by CouleeApps** [comment]
Reproduces for me on dev.

**Comment by CouleeApps** [comment]
Looks like the token object it's trying to convert is invalid: 
<img width="510" alt="image" src="https://user-images.githubusercontent.com/711973/216694696-2b200b31-0b62-4a0a-b0b5-328cdb6dbd76.png">

**Comment by CouleeApps** [comment]
The underlying bug is: the default constructor for InstructionTextToken does not initialize its fields properly, so you get garbage stack memory in there. Then, PySide tries (and fails) to convert the token into a python object, and gets real confused.
https://github.com/Vector35/binaryninja-api/blob/5bd02a445a8f7e4d560175ed6cdc35458ad78e7a/architecture.cpp#L52-L54

The fix is: we need to initialize those fields. Also I'm adding a PyErr_Print() call to the bindings when the conversion fails so we can know why, if it fails, in the future. Also I am going to add try/except guards around the scripting console magic vars so these errors won't prevent console use.

Good find on the testcase, I am surprised this hasn't happened more.

**Comment by CouleeApps** [comment]
This will be fixed in 3.3.4035-dev
